demo/data.ts

```diff
@@ -11,51 +12,54 @@ export interface IDataMock {
 export const generateMockData = (rows: number) => unpackIntoColumnsAndData([
     {
         id: 'rows',
-        type: 'numeric',
+        type: ColumnType.Number,
         editable: false,
         data: gendata(i => i, rows)
     },
 
     {
         id: 'ccc',
         name: ['City', 'Canada', 'Toronto'],
-        type: 'numeric',
+        type: ColumnType.Number,
         data: gendata(i => i, rows)
     },
 
     {
         id: 'ddd',
         name: ['City', 'Canada', 'Montréal'],
-        type: 'numeric',
+        type: ColumnType.Number,
         data: gendata(i => i * 100, rows)
     },
 
     {
         id: 'eee',
         name: ['City', 'America', 'New York City'],
-        type: 'numeric',
+        type: ColumnType.Number,
         data: gendata(i => i, rows)
     },
 
     {
         id: 'fff',
         name: ['City', 'America', 'Boston'],
-        type: 'numeric',
+        type: ColumnType.Number,
         data: gendata(i => i + 1, rows)
     },
 
     {
         id: 'ggg',
         name: ['City', 'France', 'Paris'],
-        type: 'numeric',
+        type: ColumnType.Number,
         editable: true,
         data: gendata(i => i * 10, rows)
     },
 
     {
         id: 'bbb',
         name: ['', 'Weather', 'Climate'],
-        type: 'dropdown',
+        type: ColumnType.Text,
+        text: {
+            presentation: 'dropdown'
+        },
```

Comment:
The dropdown type does not exist anymore, instead it is managed through the type's configuration nested prop -- for text the presentation options are 'input' and 'dropdown' at the moment